examples/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "zenoh-flow-examples"
+version.workspace = true
+repository.workspace = true
+homepage.workspace = true
+categories.workspace = true
+description.workspace = true
+edition.workspace = true
+license.workspace = true
+readme = "README.md"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[profile.dev]
+debug = true
+
+[profile.release]
+debug = true
+
+[dev-dependencies]
+async-std = "1.12"
```

Comment:
```suggestion
async-std = { workspace = true }
```

Reply:
That's because you are not aligned with the latest changes on master.
The latest commit introduced them as workspace dependencies.